### DISCUSSION TITLE
formula for DT score increase based on speed increase

### DISCUSSION THREAD

**Discussion by jojopipe** [ppy/osu#19322] [Ideas]
In a recent update this was already changed, so the score increase is automatically x1.00 if the speed increase isn't x1.5. I think however that this isn't the perfect solution to address this issue. If the speed increase is x1.25 the score multiplier should be x1.06 instead of  x1.00. I have regressed a formula to match the following values:
spd--------score
1------------1
1.5---------1.12
1.25--------1.06
2-----------1.24

formula: **score increase = 0.24 * speed increase + 0.76**

Seeing this implemented in the game would be really cool I think, since it would also help with the scoreboards.

**Comment by frenzibyte** [comment]
A linear scale may not be the best approach for rate-adjust mods, that's why removing multipliers from all mods with custom settings was done in the first place (as a first step, not permanent).
